retreaver-typescript/src/Campaign.ts

```
@@ -0,0 +1,398 @@
+  }
+
+  /**
+   * Set number matching tags (with validation)
+   * @param tags - Tags to set
+   * @returns Validated tags object
+   */
+  setNumberMatchingTags(tags: TagCollection | string): TagCollection {
+    if (typeof tags === 'string') {
+      tags = RetreaverNumber.extractTagsFromString(tags);
+    }
+    if (tags && (typeof tags === "object") && !(tags instanceof Array)) {
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)